foreman_salt.gemspec

```diff
@@ -14,5 +14,5 @@ Gem::Specification.new do |s|
   s.test_files = Dir['test/**/*']
 
   s.add_dependency 'deface', '< 1.0'
-  s.add_dependency 'foreman-tasks', '~> 0.6.9'
+  s.add_dependency 'foreman-tasks', '~> 0.7.1'
```

Comment:
You'd probably want to still accept 0.6.x, e.g. https://github.com/theforeman/foreman_chef/pull/23/files